Apps/phironnet/ironnet.json

```diff
@@ -1050,6 +1050,123 @@
+                    "data_type": "string",
+                    "required": false,
+                    "order": 2
+                },
+                "status": {
+                    "description": "The statuses of the IronDefense alerts to filter. Enter in CSV format.",
+                    "data_type": "string",
+                    "required": false,
+                    "order": 3
+                },
+                "min_severity": {
+                    "description": "The minimum severity of IronDefense alerts to filter.",
+                    "data_type": "numeric",
+                    "required": false,
+                    "order": 6
+                },
+                "max_severity": {
+                    "description": "The maximum severity of IronDefense alerts to filter.",
+                    "data_type": "numeric",
+                    "required": false,
+                    "order": 7
+                }
+            },
+            "output": [
+                {
+                    "data_path": "action_result.status",
+                    "data_type": "string",
+                    "column_name": "Status",
+                    "column_order": 0,
+                    "example_values": [
+                        "success",
+                        "failed"
+                    ]
+                },
+                {
+                    "data_path": "action_result.parameter.alert_id",
+                    "data_type": "string",
+                    "column_name": "Alert IDs",
+                    "column_order": 1
+                },
+                {
+                    "data_path": "action_result.parameter.category",
+                    "data_type": "string",
+                    "column_name": "Alert Categories",
+                    "column_order": 2
+                },
+                {
+                    "data_path": "action_result.parameter.sub_category",
+                    "data_type": "string",
+                    "column_name": "Alert SubCategories",
+                    "column_order": 3
+                },
+                {
+                    "data_path": "action_result.parameter.status",
+                    "data_type": "string",
+                    "column_name": "Alert Status",
+                    "column_order": 4
+                },
+                {
+                    "data_path": "action_result.parameter.min_severity",
+                    "data_type": "numeric",
+                    "column_name": "Alert Severity Minimum",
+                    "column_order": 5
+                },
+                {
+                    "data_path": "action_result.parameter.max_severity",
+                    "data_type": "numeric",
+                    "column_name": "Alert Severity Maximum",
+                    "column_order": 6
+                },
+                {
+                    "data_path": "action_result.data",
+                    "data_type": "string"
```

Comment:
We should fill out the contents of the data in this section - phantom includes a script on the VM that helps you get started.  

 * Find an example "get alerts" action you performed in your phantom and find the app run ID of the playbook execution
 * Copy the current version of ironnet.json over to your phantom VM
 * execute python /opt/phantom/bin/create_output.pyc -n <app run id of executed playbook> -u <username> -p <password> -j /path/to/ironnet.json
 * retreive updated output JSON from ironnet.json file
 * Go through and clean up the output that it generates (using common example values, etc..